packages/site-admin/CHANGELOG.md

```diff
@@ -2,25 +2,27 @@
 
 Initial release of the site-admin package providing a framework for building modern WordPress admin interfaces.
 
-## Core Components
+### Components
 
 - `SidebarButton`: Base button component for sidebar interactions
 - `SidebarContent`: Main sidebar layout component with navigation context
 - `SidebarNavigationItem`: Interactive navigation items with icon support
 - `SidebarNavigationScreen`: Screen container for sidebar navigation
 - `SiteIcon`: Site icon display component
 
-## Routing System
+### Routing System
 - `RoutesContext`: Context provider for current matched route
 - `ConfigContext`: Context for router configuration and settings
 - `browserHistory`: Browser history instance for navigation state
 
-### Router Hooks
+#### Router Hooks
 - `useHistory`: Hook for navigation and history management
 - `useLink`: Hook for handling link navigation
 - `useLocation`: Hook for accessing current route location
 - `useMatch`: Hook for route matching functionality
 
 ## Next
```

Comment:
Not related to this PR specifically, but I'm not sure I get the sense for the "Next" section?

I guess the README should always reflect a snapshot of the codebase, and we could otherwise have a CHANGELOG (or other ways) to track releases and what they included?

Reply:
To clarify, this is the CHANGELOG.md file, not the README.md

Reply:
🤦  Got it. I got mixed up.